versioning.md

```diff
@@ -0,0 +1,29 @@
+Versioning our libraries
+========================
+
+Which repositories?
+-------------------
+
+Some of our repositories are used as libraries and included as a requirement in other repositories.
+eg https://github.com/OpenDataServices/flatten-tool/releases
+
+For these libraries, we want to version them to make it easier to do this.
```

Comment:
I think that docs should state what we do, not what we want to do. So, we should say "For these libraries, we version them to make it easier to do this. This applies to all new libraries. As of July 2018, work to version existing libraries is still in progress".

Reply:
I'm not sure we have fully agreed about this - and I note your later comment talks about User needs, which might contradict this. Can we leave this pull request at it is (just saying "some", which is technically true) and have some Loomio discussion to clarify and make more specific points like this later?